### DISCUSSION TITLE
Zephyr support for STM32U5 series MCU

### DISCUSSION THREAD

**Discussion by 99003662** [zephyrproject-rtos/zephyr#48333] [Q&A]
Hi team,
            I am working with STM32U5 series board, I have couple of question here 
1. Currently in zephyr , what all are the features and peripheral support is available for this particular series(STM32U5) , 
2. Do we have an estimated timeline by which zephyr will completely support the stm32u5 family. 

 Based on these , we need to take critical decision so , kindly help me here .
Thanks

**Comment by erwango** [comment]
For 1: See response to the same question here:
https://github.com/zephyrproject-rtos/zephyr/discussions/48028#discussioncomment-3232509

**Comment by erwango** [comment]
For 2., Not all STM32U5 hw are supported by Zephyr API yet (and this might be the case for years). So it's a tricky point to anwser about complete support is a vague. But it is also unlikely you actually need support for ALL U5 HW features.
Please be specific on the HW support you actually need and we might be able to provide more detailed answer.

So it depends of the features you actually need.
